cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,37 @@
 /// <reference types='cypress' />
 
+import { user } from '../support/testData';
+import { book } from '../support/testData';
+
 describe('Book Store app', () => {
   before(() => {
-    
+    cy.visit('/');
   });
 
-  it('', () => {
```

Comment:
Consider splitting the test flow into three separate tests for better organization and readability. This will also make it easier to identify which part of the test is causing issues if a test fails.